target/scripts/startup/setup.d/security/misc.sh

```diff
@@ -186,8 +186,18 @@ function __setup__security__clamav
     if [[ ${CLAMAV_MESSAGE_SIZE_LIMIT} != '25M' ]]
     then
       _log 'trace' "Setting ClamAV message scan size limit to '${CLAMAV_MESSAGE_SIZE_LIMIT}'"
-      sedfile -i \
-        "s/^MaxFileSize.*/MaxFileSize ${CLAMAV_MESSAGE_SIZE_LIMIT}/" \
+
+      # do a short sanity checks; ClamAV stops scanning at more that 4GB file size
+      if [[ $(numfmt --from=si "${CLAMAV_MESSAGE_SIZE_LIMIT}") -gt $(numfmt --from=si 4G) ]]
```

Comment:
sorry, for catching this now. i think clamav is not using the SI system for their limit variables.

i think the M / G in the clamav variables is actually MiB / GiB, so this check is not correct as it checks for 4000000000 instead of 4294967296.

Could it be that the correct check would involve:
```
if [[ $(numfmt --from=iec "${CLAMAV_MESSAGE_SIZE_LIMIT}") -gt $(numfmt --from=iec 4G) ]]
```